test.md

```diff
@@ -0,0 +1 @@
+## This file should be ignored by git
```

Comment:
- [ ] This file is actually not being ignored by git. To make this work properly, you need to add a `.gitignore` file to your project. Inside that file, you can type `test.md` so that the file with that name is ignored.

![Captura de Pantalla 2022-09-20 a la(s) 9 58 15 a  m](https://user-images.githubusercontent.com/86022690/191292744-fbbb128d-a264-4601-a9c0-6563a5ffe5cf.png)